src/components/leaderboards/Leaderboard/index.js

```diff
@@ -20,11 +21,12 @@ module.exports = React.createClass({
     charitySlug: React.PropTypes.string,
     charityUid: React.PropTypes.string,
     country: React.PropTypes.oneOf(['au', 'ie', 'nz', 'uk', 'us']),
-    type: React.PropTypes.oneOf(['team', 'individual']),
+    type: React.PropTypes.oneOf(['individual']),
```

Comment:
The type is used within the Leaderboard mixin to determine the correct endpoint to hit.